module/VuFind/src/VuFind/Config/Locator.php

```diff
@@ -83,21 +105,31 @@ public static function getBaseConfigPath($filename, $path = 'config/vufind')
     /**
      * Get the file path to a config file.
      *
-     * @param string $filename config file name
-     * @param string $path     path relative to VuFind base (optional; defaults
+     * @param string  $filename Config file name
+     * @param ?string $path     Path relative to VuFind base (optional; defaults
```

Comment:
Maybe change the comment to say "use null for default" since that behavior has changed slightly?